tools/api-markdown-documenter/api-report/api-markdown-documenter.beta.api.md

```diff
@@ -61,6 +60,7 @@ export { ApiItemKind }
 // @public
 export interface ApiItemTransformationConfiguration extends ApiItemTransformationConfigurationBase, DocumentationSuiteConfiguration, Required<LoggingConfiguration> {
     readonly defaultSectionLayout: (apiItem: ApiItem, childSections: Section[] | undefined, config: ApiItemTransformationConfiguration) => Section[];
+    readonly startingHeadingLevel: number;
```

Comment:
Note for reviewers: since we now "normalize" the markdown output as a part of the initial transformation step, we have to resolve heading levels in this step. This config option was moved from the `toMarkdown` and `toHtml` steps and now occurs here.

Reply:
Note that users still write API item transformations in terms of the pre-normalized markdown, so they still don't have to worry about reasoning about section hierarchy.